Scoop/NetworkManager.swift

```diff
@@ -21,4 +21,32 @@ class NetworkManager {
             ]
             return headers
         }
+    
+    static func searchLocation(depatureDate: String, startLocation: String, endLocation: String, completion: @escaping (RideResponse) -> Void) {
+        let endpoint = "https://scoop-prod.cornellappdev.com/api/search/"
```

Comment:
Maybe use the hostEndpoint variable rather than the whole URL, since the variable is connected to the Keys.
Not sure if we should have the URL in the code straight up

Reply:
Yup going to fix that, thanks for pointing that out! When networking was broken at first, I had thought maybe the URL was the reason why hence ^